docs/source/conf.py

```diff
@@ -3557,6 +3557,15 @@
 
 from sphinx.ext.coverage import CoverageBuilder
 
+# NB: Due to some duplications of the following modules/functions, we keep
```

Comment:
Why not just de-duplicate them in the .rst file?

Reply:
We removed the modules in the previous PR but that breaks the coverage check.  The PR title and description are a bit of a carry over from the first PR
Actually can you shed any light on https://github.com/pytorch/pytorch/pull/123244#issuecomment-2037681048?  We're pretty sure this is the problem but we don't know how to resolve this or even what the intended behavior is

Reply:
Also, this issue has been around for a while but never surface before because running sphinx in parallel hide it https://github.com/sphinx-doc/sphinx/issues/4459.  Maybe it's time to try a newer Sphinx version.  I'm also curious what will happen when we have both docs for `torch.ao.quantization.quantize` as a module and `torch.ao.quantization.quantize` as a function.